docs/en/interfaces/http.md

```diff
@@ -205,7 +205,13 @@ For information about other parameters, see the section "SET".
 
 Similarly, you can use ClickHouse sessions in the HTTP protocol. To do this, you need to add the `session_id` GET parameter to the request. You can use any string as the session ID. By default, the session is terminated after 60 seconds of inactivity. To change this timeout, modify the `default_session_timeout` setting in the server configuration, or add the `session_timeout` GET parameter to the request. To check the session status, use the `session_check=1` parameter. Only one query at a time can be executed within a single session.
 
-You have the option to receive information about the progress of query execution in X-ClickHouse-Progress headers. To do this, enable the setting send_progress_in_http_headers.
+You have the option to receive information about the progress of query execution in `X-ClickHouse-Progress` headers. To do this, enable the setting [send_progress_in_http_headers](../operations/settings/settings.md#settings-send_progress_in_http_headers). Example of the header:
+
+```
+X-ClickHouse-Progress: {"read_rows":"393216","read_bytes":"28368934","total_rows":"8880128"}
```

Comment:
probably the example could demonstrate how these values change over time for a single query